Hero

OlesiaZaozirna opened the pull request.

OlesiaZaozirna committed:
update bestsellers section

OlesiaZaozirna committed:
Merge branch 'main' of https://github.com/Serhieie/codecrafters-project

OlesiaZaozirna committed:
update hero

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  color: var(--blue-font);
+  font-family: Playfair Display;
+  font-size: 28px;
+  font-style: normal;
```
зайві параметри

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  font-family: Playfair Display;
+  font-size: 28px;
+  font-style: normal;
+  font-weight: 400;
```
-

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  font-size: 28px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.5;
```
-

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+.hero-text {
+  color: var(--blue-font);
+  font-family: Roboto;
+  font-size: 16px;
```
фонт сайз 16 задан на боді по замовчуванню з мобілки

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  color: var(--blue-font);
+  font-family: Roboto;
+  font-size: 16px;
+  font-style: normal;
```
-

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  font-family: Roboto;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
```
-

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.5;
```
-

Serhieie commented on src/css/hero.css:
```diff
@@ -0,0 +1,166 @@
+  }
+  .hero-text {
+    max-width: 312px;
+    font-size: 18px;
```
з 768 у медіа для боді задан розмір 18px

OlesiaZaozirna committed:
update hero.css

1. Підвищення спецефічності  для классу .hero-container бо перебивало стилі при залитті на гітхаб.
2. Видалення дублюющихся силів, які прописанні у root
